lib/pulfalight/traject/ead2_component_config.rb

```diff
@@ -457,14 +457,30 @@
 to_field "geogname_ssm", extract_xpath("./controlaccess/geogname")
 to_field "places_ssim", extract_xpath("./controlaccess/geogname")
 
-to_field "access_subjects_ssim", extract_xpath("./controlaccess", to_text: false) do |_record, accumulator|
-  accumulator.map! do |element|
+to_field "access_subjects_ssim" do |record, accumulator|
```

Comment:
We got some more info on these fields? Curious what you found out.

Reply:
They are used in the generic text search as well as returned hit highlighting.

Reply:
And they'll include values we want to replace?